NSBrazilConf/Views/Home/Feed/PastEditionCardView.swift

```diff
@@ -12,33 +12,43 @@ struct PastEditionCardView: View {
     let video: Video
     
     var body: some View {
-        VStack(alignment: .center) {
+        VStack(alignment: .leading, spacing: 10) {
             Text(video.title)
                 .font(.title)
                 .fontWeight(.bold)
                 .foregroundColor(Color.white)
-                .padding(.top, 24)
-                .padding()
+                .padding(.horizontal, 10)
+                .padding(.top, 10)
                 .lineLimit(4)
-                .frame(width: 240)
+                .shadow(radius: 1)
             Text(video.speaker)
                 .font(.headline)
                 .fontWeight(.bold)
                 .foregroundColor(Color.white)
-                .padding()
+                .padding(.horizontal, 10)
                 .lineLimit(1)
+                .shadow(radius: 1)
             Spacer()
-            Image("ic_play")
+            HStack {
+                Spacer()
+                Image("ic_play")
                 .resizable()
                 .renderingMode(.original)
                 .aspectRatio(contentMode: .fit)
                 .frame(width: 96, height: 96)
                 .padding()
+                Spacer()
+            }
             Spacer()
+            .fixedSize()
+            .frame(width: nil, height: 30)
             
         }
+        .background(
+            ImageViewContainer(imageURL: video.background, hasPadding: false, contentMode: .fill)
+        )
         .background(video.backgroundColor)
-        .cornerRadius(30)
+        .cornerRadius(15)
         .frame(width: 246, height: 360)
         .shadow(color: Color.gray.opacity(0.5), radius: 10, x: 0, y: 0)
```

Comment:
Consegue alterar para onde for sombra e usar `Color.gray`, usar o `Color.black`? Pq quando estiver no modo darkmode nem precisa tratar a sombra, ela já desaparece automaticamente. Com o gray, fica uma sombrinha chata. Eu deveria ter feito. 

 `shadow(color: Color.black.opacity(0.2), radius: 10, x: 0, y: 0)`

Reply:
Mudo sim! Eu nem cheguei a testar com dark mode pq a gente tá com uma task só pra isso, mas é melhor que esse elemento já fica certinho :D

Reply:
Acho que seria interessante ter uma CardView que recebe uma view como parâmetro no init para que não seja necessário aplicar o efeito de sombra em todos os cards